docs/Manual Installation.md

```diff
@@ -31,23 +31,14 @@ You have a few options for installing RITA.
 
 ##### Installing Golang
 
-In order to compile RITA manually you will need to install both [Golang](https://golang.org) and [Dep](https://github.com/golang/dep).
```

Comment:
Using go modules means we can remove a bunch of the steps that used to be necessary. This should get new contributors up to speed much easier and with fewer problems.